manage.py

```diff
@@ -63,6 +63,28 @@ def runserver(info, host, port, debugger):
                extra_files=reload_on_files)
 
 
+@cli.command()
+def check_highlevel():
+    try:
+       h = db.data.check_highlevel()
+       h_meta = db.data.check_highlevel_meta()
+       print("The no of submissions without highlevel metadata :",
```

Comment:
I would like to see more information here. It's not very useful to only see the number of failed highlevel calculations, I would also like to know what MBIDs failed, and be able to download the lowlevel data file to experiment with it.
This means that this method should not only print the count, but the actual items themselves. This could include: 1) the highlevel.id, 2) the mbid (lowlevel.gid), 3) the offset (important if there have been more than 1 submission for an mbid so that we know which one do download)

It would be good to have this method able to show a count, and then the actual failed items when a flag is set, e.g.:

    $ manage.py highlevel failed_rows
    Number of highlevel rows that failed processing: 5

    $ manage.py highlevel failed_rows --detail
    Number of highlevel rows that failed processing: 5
    id, mbid, number
    3, 65e291fc-1bfa-4f06-a7e3-767b5951454b, 3
    ...

Note that the message describes the reason that we don't have a highlevel_meta row (failed processing), rather than just describing it. I think that it's a nice idea to make this more explicit.